Telegram/SourceFiles/mainwindow.cpp

```diff
@@ -166,14 +166,14 @@ void MainWindow::firstShow() {
 		: lng_enable_notifications_from_tray);
 
 	if (isLinux) {
-		trayIconMenu->addAction(lang(lng_open_from_tray), this, SLOT(showFromTray()))->setEnabled(true);
+		trayIconMenu->addAction(lang(lng_open_from_tray) + QCoreApplication::applicationName(), this, SLOT(showFromTray()))->setEnabled(true);
```

Comment:
That's not a good idea. There're languages that use a different word order (e.g. in Azerbaijani that should be something like "Kepka/Telegram çıxın"). I suggest that we use the text templates here (e.g. "Quit {app_name}", like [here](https://github.com/minlexx/kepka/blob/34e3ac75e189da9bf84cd0588cd40afbf2185234/Telegram/Resources/langs/lang.strings#L99)) and use string format to prepare the resulting value.

Reply:
Yep, will fix later when I have time. Need to figure out how this translation system works... I see it automatically generates some .cpp and .h files